src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,8 @@ public class TypicalIndexes {
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+
+    public static final Index INDEX_FIRST_EVENT = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_EVENT = Index.fromOneBased(2);
+    public static final Index INDEX_THIRD_EVENT = Index.fromOneBased(3);
```

Comment:
Do you think it would be better to combine these variables with `INDEX_X_PERSON` variables since we are using one based for both Persons and Events?